content-src/components/DiscoveryStreamBase/_DiscoveryStreamBase.scss

```diff
@@ -27,4 +27,11 @@
     display: grid;
     grid-row-gap: var(--gridRowGap);
   }
+
+  .ds-header {
```

Comment:
I moved this into layout generic CSS, as it's not reusable. (We could make it into a component if it becomes more than just a className)